### PR TITLE
V5.31 documentation

### PR DESCRIPTION
**Pull request by amyblais**
Needs one approval to be able to merge.

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,80 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
 
+## Release v5.31 - [ESR](https://docs.mattermost.com/administration/extended-support-release.html)
+
+**Release Day: 2021-01-16**
+
+Mattermost v5.31.0 contains a low level security fix. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) is recommended. Details will be posted on our [security updates page](https://mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://mattermost.org/responsible-disclosure-policy/).
+
+### Compatibility
+
+ - Support for Mattermost Server v5.25 [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```
Comment (justinegeffen) [inline source/administration/changelog.md:15]:
```suggestion
 - Support for Mattermost Server v5.25 [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.30 or later is highly recommended.
```

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,80 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+
+### Highlights
+
+#### Improved status updates for [Incident Management (E20 Edition)](https://mattermost.com/incident-management/)
+ - Pre-packaged and pre-installed the Mattermost Incident Management v1.2.0, which enables incident responders to more easily inform stakeholders of incident status updates.
```
Comment (justinegeffen) [inline source/administration/changelog.md:20]:
```suggestion
 - Pre-packaged and pre-installed the Mattermost Incident Management v1.2.0, which enables incident responders to easily inform stakeholders of incident status updates.
```

Comment (cwarnermm) [inline source/administration/changelog.md:20]:
```suggestion
 - Pre-packaged and pre-installed the Mattermost Incident Management v1.2.0, which enables incident responders to inform stakeholders more easily of incident status updates.
```

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,80 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Issuing ``/dnd`` consecutively no longer unexpectedly toggles the user's status between “Do Not Disturb” and “Online”, and will only set the user’s status to “Do Not Disturb”.
+
+#### Administration
+ - Added a new ``manage_remote_clusters`` permission.
+
+### Bug Fixes
+ - Fixed an issue where a column in the ``ThreadMemberships`` table was missing.
+ - Cleaned up the config store on server initialization errors.
+ - Fixed an issue where permissions did not grant read and/or write access to the Global Relay configuration settings.
+ - Fixed an issue where the site configuration ‘’Read only’’ permission did not make the "Notice" section read-only for the System Manager.
```
Comment (justinegeffen) [inline source/administration/changelog.md:37]:
```suggestion
 - Fixed an issue where the site configuration "Read only" permission did not make the "Notice" section read-only for the System Manager.
```

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,80 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Fixed an issue where importing Client4 in a node server caused an exception due to rudder modules.
+ - Fixed an issue where LDAP ‘’FirstLoginSync’’ didn't close the LDAP Session.
```
Comment (justinegeffen) [inline source/administration/changelog.md:39]:
```suggestion
 - Fixed an issue where LDAP "FirstLoginSync" didn't close the LDAP session.
```

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,6 +4,80 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Fixed an issue where line numbers did not line up with the text on code file previews.
+ - Fixed an issue where the threshold from the bottom of the screen was sometimes not respected for received messages.
+ - Fixed an issue where desktop notifications got sent for every message posted in a Direct Message channel.
+
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in config.json:
```
Comment (justinegeffen) [inline source/administration/changelog.md:48]:
```suggestion
 - Under ``ServiceSettings`` in ``config.json``:
```

File: source/administration/extended-support-release.rst [hunk 6]
```diff
@@ -6,7 +6,7 @@ What is an Extended Support Release?
 During each monthly release, Mattermost backports high severity or high impact security fixes to the previous three monthly releases. Extended Support Releases (ESRs) are releases that will receive backports for security fixes and major bug fixes for the length of their life cycle.
 
 .. important::
-  Support for server `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ (ESR) 5.19 has come to the end of its lifecycle. Upgrading to server v5.25 or later is required.
+  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```
Comment (justinegeffen) [inline source/administration/extended-support-release.rst:9]:
```suggestion
  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.30 or later is highly recommended.
```

File: source/administration/important-upgrade-notes.rst [hunk 7]
```diff
@@ -2,10 +2,10 @@ Important Upgrade Notes
 =======================
 
 .. important::
-   PostgreSQL ended long-term support for `version 9.4 in February 2020 <https://www.postgresql.org/support/versioning>`_. From v5.26 Mattermost officially supports PostgreSQL version 10 as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16). Please follow the instructions under the Upgrading Section within `the PostgreSQL documentation <https://www.postgresql.org/support/versioning/>`_.
-   
+   Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```
Comment (justinegeffen) [inline source/administration/important-upgrade-notes.rst:5]:
```suggestion
   Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.30 or later is highly recommended.
```

File: source/administration/release-lifecycle.rst [hunk 8]
```diff
@@ -6,7 +6,7 @@ What is a Release Lifecycle?
 During each monthly release, Mattermost backports high severity or high impact security fixes to the previous three monthly releases. `Extended Support Releases <https://docs.mattermost.com/administration/extended-support-release.html>`_ receive backports for security fixes and major bug fixes for the length of their life cycle.  Once a release life cycle has ended, it will no longer receive the security or major bug backports. We follow the same release cycle pattern for all upcoming releases.
 
 .. important::
-  Support for server `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ (ESR) 5.19 has come to the end of its lifecycle. Upgrading to server v5.25 or later is required.
+  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```
Comment (justinegeffen) [inline source/administration/release-lifecycle.rst:9]:
```suggestion
  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.30 or later is highly recommended.
```

File: source/administration/mmctl-cli-tool.rst [hunk 9]
```diff
@@ -2355,36 +2356,65 @@ mmctl permissions add
...
 
-mmctl permissions role assign
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+mmctl permissions remove
+^^^^^^^^^^^^^^^^^^^^^^^^
 
 **Description**
 
-Assign users to a role by username (only available in Enterprise Edition E10 and E20).
+  Remove one or more permissions from an existing role (only available in E10 and E20).
```
Comment (cwarnermm) [inline source/administration/mmctl-cli-tool.rst:2364]:
```suggestion
  Remove one or more permissions from an existing role (available in Enterprise Edition E10 and E20).
```

File: source/administration/upgrade.rst [hunk 10]
```diff
@@ -5,7 +5,7 @@ In most cases you can upgrade Mattermost Server in a few minutes, but the upgrad
 
 .. important::
 
-  Support for server `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ (ESR) 5.19 has come to the end of its lifecycle. Upgrading to server v5.25 or later is required.
+  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```
Comment (justinegeffen) [inline source/administration/upgrade.rst:8]:
```suggestion
  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.30 or later is highly recommended.
```

File: source/administration/version-archive.rst [hunk 11]
```diff
@@ -4,7 +4,7 @@ Version Archive
 If you want to check that the version of Mattermost you are installing is the official, unmodified version, compare the SHA-265 checksum or the file's GPG signature with the one published in this version archive. To verify the GPG signature of a Mattermost release, use the public key stored in Keybase: https://keybase.io/mattermost.
 
 .. important::
-  Support for server `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ (ESR) 5.19 has come to the end of its lifecycle. Upgrading to server v5.25 or later is required.
+  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```
Comment (justinegeffen) [inline source/administration/version-archive.rst:7]:
```suggestion
  Support for Mattermost Server v5.25 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.30 or later is highly recommended.
```

File: source/help/getting-started/organizing-conversations.rst [hunk 12]
```diff
@@ -154,7 +154,9 @@ When configured by the System Admin, users can enable the features in **Account
 
 **Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.
```
Comment (justinegeffen) [inline source/help/getting-started/organizing-conversations.rst:155]:
```suggestion
**Collapsible custom categories:** Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.
```

File: source/help/getting-started/organizing-conversations.rst [hunk 13]
```diff
@@ -154,7 +154,9 @@ When configured by the System Admin, users can enable the features in **Account
...
 
-**Drag and drop channels and categories**: Drag selected channels between or within categories, or drag to reorder entire categories to prioritize important conversations. You can also specify a destination for selected channels using the **Move to** option under the **More options** icon. The **Move to** option is available today for Cloud deployments and will be available for self-hosted deployments in Mattermost Server v5.32 and later.
+**Using the Mattermost Web or Desktop App, select multiple channels at a time. To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux). To clear channel selections, press ESC.
```
Comment (justinegeffen) [inline source/help/getting-started/organizing-conversations.rst]:
```suggestion
**Select multiple channels at a time in the Mattermost Web or Desktop App:** To select sequential channels, press and hold SHIFT. To select non-sequential channels press and hold CMD (for Mac) or CTRL (for Windows/Linux). To clear channel selections, press ESC.
```

File: source/help/getting-started/organizing-conversations.rst [hunk 14]
```diff
@@ -154,7 +154,9 @@ When configured by the System Admin, users can enable the features in **Account
...
+
+**Drag and drop channels and categories**: Drag selected channels between or within categories, or drag to reorder entire categories to prioritize important conversations. Multi-selected channels move together as a group in the order they originally appeared. You can also specify a destination for selected channels using the **Move to** option under the **More options** icon. The **Move to** option is available today for Cloud deployments and will be available for self-hosted deployments in Mattermost Server v5.32 and later.
```
Comment (justinegeffen) [inline source/help/getting-started/organizing-conversations.rst:161]:
```suggestion
**Drag and drop channels and categories:** Drag selected channels between or within categories, or drag to reorder entire categories to prioritize important conversations. Multi-selected channels move together as a group in the order they originally appeared. You can also specify a destination for selected channels using the **Move to** option under the **More options** icon. The **Move to** option is available today for Cloud deployments and will be available for self-hosted deployments in Mattermost Server v5.32 and later.
```

**Review by cwarnermm — APPROVED** [review]
Very minor suggested updates provided inline.

File: source/help/getting-started/organizing-conversations.rst (hunk 12 above)
Comment (cwarnermm) [reply source/help/getting-started/organizing-conversations.rst:155]: Great catch, @justinegeffen!

File: source/help/getting-started/organizing-conversations.rst (hunk 13 above)
Comment (cwarnermm) [reply source/help/getting-started/organizing-conversations.rst]: I applied this change as part of a merge conflict resolution.

File: source/help/getting-started/organizing-conversations.rst (hunk 14 above)
Comment (cwarnermm) [reply source/help/getting-started/organizing-conversations.rst:161]: Another great catch, @justinegeffen! Thanks!

File: source/administration/changelog.md (hunk 1 above)
Comment (amyblais) [reply source/administration/changelog.md:15]: @justinegeffen v5.31 is the new ESR, and this is the reason why we're recommending upgrading to this version.

Comment (amyblais) [reply source/administration/changelog.md:15]: I'll clarify the docs.

**Pull request merged**